Non-standard character in Q22 results in build failure

```
$ ant build
Buildfile: /root/oltpbench/build.xml

build:
    [javac] Compiling 419 source files to /root/oltpbench/build
    [javac] /root/oltpbench/src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java:72: error: unmappable character for encoding ASCII
    [javac]         // I1 ??? I7 are randomly selected without repetition from the possible values
    [javac]               ^
    [javac] /root/oltpbench/src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java:72: error: unmappable character for encoding ASCII
    [javac]         // I1 ??? I7 are randomly selected without repetition from the possible values
    [javac]                ^
    [javac] /root/oltpbench/src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java:72: error: unmappable character for encoding ASCII
    [javac]         // I1 ??? I7 are randomly selected without repetition from the possible values
    [javac]                 ^
    [javac] 3 errors

BUILD FAILED
/root/oltpbench/build.xml:76: Compile failed; see the compiler error output for details.

Total time: 1 second
```

Looking into src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java:72, I find
`// I1 … I7 are randomly selected without repetition from the possible values`, which `…` is not a standard character. I suggest changing the character to `...`

--- a/src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java
+++ b/src/com/oltpbenchmark/benchmarks/tpch/procedures/Q22.java
@@ -69,7 +69,7 @@ public class Q22 extends GenericQuery {
 
     @Override
     protected PreparedStatement getStatement(Connection conn, RandomGenerator rand) throws SQLException {
-        // I1 … I7 are randomly selected without repetition from the possible values
+        // I1 ... I7 are randomly selected without repetition from the possible values
         // for Country code as defined in Clause 4.2.2.9
 
         // We are given

